libr/arch/p/bpf_cs/plugin.c

```diff
@@ -4,6 +4,7 @@
 #include <r_esil.h>
 #include <r_lib.h>
 
+#define CAPSTONE_AARCH64_COMPAT_HEADER
```

Comment:
is this necessary for bpf? same for the other archs

Reply:
Will try to delete it whenever possible